Include/compile.h

```diff
@@ -102,4 +102,7 @@ PyAPI_FUNC(int) _PyAST_Optimize(struct _mod *, PyArena *arena, _PyASTOptimizeSta
 #define Py_eval_input 258
 #define Py_func_type_input 345
 
+/* This doesn't need to match anything */
+#define Py_fstring_input 0
```

Comment:
Well, I guess it specifically must match nothing else? And zero is an easy value to get by accident. Why not make it something way out of range?